.circleci/config.yml

```diff
@@ -206,6 +206,6 @@ workflows:
             - build
           filters:
             branches:
-              only: /^(?!pull\/).*$/ # don't deploy to gh pages on PRs.
               ignore:
                 - ^dependabot/.*/
+                - ^pull/.*/ # don't deploy to  gh pages on PRs.
```

Comment:
I think these aren't being parsed as regexes so they aren't matching any branches, since they're missing the leading `/`:
```suggestion
                - /^dependabot/.*/
                - /^pull/.*/ # don't deploy to  gh pages on PRs.
```

Reply:
But I do like the simplified config with only an ignore section